Sources/Performance/main.swift

```diff
@@ -89,8 +90,16 @@ do {
     server.drain { client in
         let parser = HTTP.RequestParser()
         let serializer = HTTP.ResponseSerializer()
+        let websocketMiddleware = WebSocketMiddleware(client: client)
+        
+        websocketMiddleware.onConnect { websocket in
+            websocket.textStream.drain { string in
+                websocket.textStream.inputStream(string)
```

Comment:
we should put a helper on input stream like `.fill` or something that matches up with `.drain` so this is more readable.